src/components/UserList/index.tsx

```diff
@@ -360,19 +368,21 @@ const UserList: React.FC = () => {
                       )}
                     </div>
                   </div>
-                  <div className="form-row">
-                    <label htmlFor="genpassword" className="checkbox-label">
-                      {intl.formatMessage(messages.autogeneratepassword)}
-                    </label>
-                    <div className="form-input">
-                      <Field
-                        type="checkbox"
-                        id="genpassword"
-                        name="genpassword"
-                        onClick={() => setFieldValue('password', '')}
-                      />
+                  {emailSettings.enabled && (
+                    <div className="form-row">
+                      <label htmlFor="genpassword" className="checkbox-label">
+                        {intl.formatMessage(messages.autogeneratepassword)}
+                      </label>
+                      <div className="form-input">
+                        <Field
+                          type="checkbox"
+                          id="genpassword"
+                          name="genpassword"
+                          onClick={() => setFieldValue('password', '')}
+                        />
+                      </div>
                     </div>
-                  </div>
+                  )}
```

Comment:
I think it's better here to just leave it disabled instead of hiding the option entirely.

Reply:
Sure, will change that!